modules/core/src/main/java/org/apache/fluo/recipes/core/map/CollisionFreeMap.java

```diff
@@ -44,12 +38,19 @@
 import org.apache.fluo.recipes.core.common.TableOptimizations.TableOptimizationsFactory;
 import org.apache.fluo.recipes.core.serialization.SimpleSerializer;
 
+import com.google.common.base.Preconditions;
```

Comment:
Sorry. Stuff came up. I'm still working on it. Hoping to have a working plugin by the end of the day.